### ISSUE TITLE
Query: Does Jazzy work with Linux version of Swift?

### ISSUE THREAD

**Issue by lloydsargent** [realm/jazzy#495]
Thanks!

**Comment by jpsim** [comment]
No, Jazzy uses SourceKit & SourceKitten for extracting documentation information from Swift source, and neither of them are available on Linux yet.

**Comment by kirbyfan64** [comment]
Both [SourceKit](https://github.com/jpsim/SourceKitten/issues/179) and [SourceKitten](https://github.com/jpsim/SourceKitten/issues/179) work on Linux. Can this be reopened?

**Comment by jpsim** [comment]
Sure.

**Comment by SDGGiesbrecht** [comment]
Much, but not all, of #487 (Use Without Xcode) is very relevant to Linux support.

@kirbyfan64, you could have a look there an see if you can get something working.

**Comment by kirbyfan64** [comment]
Looks like it's mostly usable using @jpsim's workflow in #487:

```
$ sourcekitten doc --spm-module Module > doc.json
$ jazzy ... --sourcekitten-sourcefile doc.json
```

So at this point, this is just #487 I guess? (It's worth noting anyway that Jazzy's vendored `sourcekitten` binary is for macOS, but that doesn't matter right now since it's only used for `xcodebuild` anyway...)
